README.rst

```diff
@@ -92,7 +92,7 @@ starting with version 2.0.
 
 There is a resource database implementation, ICCCM support and a
 framework for adding X extension code. Several extensions have been
-implemented; (RECORD, SHAPE, Xinerama, Composite, RANDR, and XTEST)
+implemented; (RECORD, SHAPE, Xinerama, Composite, RANDR, DPMS, and XTEST)
```

Comment:
If you'd be so glad to update change list for the next release, I'd also suggest to update this list with other extensions which were implemented previously: https://github.com/python-xlib/python-xlib/tree/master/Xlib/ext

Or I can do it when I have time to pack a release. Just leaving this comment as a reminder for me.